packages/ses/src/lockdown.js

```diff
@@ -162,28 +162,43 @@ export const repairIntrinsics = (options = {}) => {
   // for an explanation.
 
   const {
-    errorTaming = getenv('LOCKDOWN_ERROR_TAMING', 'safe'),
+    errorTaming = getenv('LOCKDOWN_ERROR_TAMING', 'safe', [
+      'unsafe',
+      'unsafe-debug',
+    ]),
     errorTrapping = /** @type {"platform" | "none" | "report" | "abort" | "exit"} */ (
-      getenv('LOCKDOWN_ERROR_TRAPPING', 'platform')
+      getenv('LOCKDOWN_ERROR_TRAPPING', 'platform', [
+        'none',
+        'report',
+        'abort',
+        'exit',
+      ])
     ),
     reporting = /** @type {"platform" | "console" | "none"} */ (
-      getenv('LOCKDOWN_REPORTING', 'platform')
+      getenv('LOCKDOWN_REPORTING', 'platform', ['console', 'none'])
     ),
     unhandledRejectionTrapping = /** @type {"none" | "report"} */ (
-      getenv('LOCKDOWN_UNHANDLED_REJECTION_TRAPPING', 'report')
+      getenv('LOCKDOWN_UNHANDLED_REJECTION_TRAPPING', 'report', ['none'])
     ),
-    regExpTaming = getenv('LOCKDOWN_REGEXP_TAMING', 'safe'),
-    localeTaming = getenv('LOCKDOWN_LOCALE_TAMING', 'safe'),
+    regExpTaming = getenv('LOCKDOWN_REGEXP_TAMING', 'safe', ['unsafe']),
+    localeTaming = getenv('LOCKDOWN_LOCALE_TAMING', 'safe', ['unsafe']),
 
     consoleTaming = /** @type {'unsafe' | 'safe'} */ (
-      getenv('LOCKDOWN_CONSOLE_TAMING', 'safe')
+      getenv('LOCKDOWN_CONSOLE_TAMING', 'safe', ['unsafe'])
     ),
     overrideTaming = /** @type {'moderate' | 'min' | 'severe'} */ (
-      getenv('LOCKDOWN_OVERRIDE_TAMING', 'moderate')
+      getenv('LOCKDOWN_OVERRIDE_TAMING', 'moderate', ['min', 'severe'])
     ),
-    stackFiltering = getenv('LOCKDOWN_STACK_FILTERING', 'concise'),
-    domainTaming = getenv('LOCKDOWN_DOMAIN_TAMING', 'safe'),
-    evalTaming = getenv('LOCKDOWN_EVAL_TAMING', 'safe-eval'),
+    stackFiltering = getenv('LOCKDOWN_STACK_FILTERING', 'concise', ['verbose']),
+    domainTaming = getenv('LOCKDOWN_DOMAIN_TAMING', 'safe', ['unsafe']),
+    evalTaming = getenv('LOCKDOWN_EVAL_TAMING', 'safe-eval', [
+      'unsafe-eval',
+      'no-eval',
+      // deprecated
+      'safeEval',
+      'unsafeEval',
+      'noEval',
```

Comment:
We should consider moving the deprecated values into another argument to automatically warn about them (and exclude from the types)

Reply:
Interesting idea about a runtime warning for the deprecated values. What I did in this PR is I continue to pass then in as the runtime behavior, but I omit them from the type.

Done